client/db.go

```diff
@@ -607,12 +611,14 @@ func (b *Batch) fillResults() error {
 				row.Key = []byte(call.Args.(*proto.PutRequest).Key)
 				if result.Err == nil {
 					row.setValue(&call.Args.(*proto.PutRequest).Value)
```

Comment:
How would DRY out the type cast?

`setTimestamp` is needed so that some tests can determine the timestamp the value was written at. See `dist_sender_server_test.go:136`.